File: GalaxyBot.Console/SlashCommands/Source.cs
namespace GalaxyBot.SlashCommands;

public class Source : InteractionModuleBase<SocketInteractionContext> {
    [SlashCommand("source", "Responds with a link to the source code of the GalaxyBot.")]
    public async Task Command() => await RespondAsync("https://www.github.com/Narolith/GalaxyBotV2");
}

File: src/cogs/source.py
from interactions import Extension, SlashContext, slash_command


class Source(Extension):
    @slash_command(name="source", description="Get a link to the bot source code.")
    async def help(self, ctx: SlashContext):
        await ctx.send("https://github.com/Stephen-Cherry/GalaxyBotv2/", ephemeral=True)
